app/frontend/src/assets/scss/style.scss

```diff
@@ -90,3 +90,41 @@ a, .v-tab {
     }
   }
 }
+
+// Stepper
+.silv-stepper {
+  .header-row{
+    background-color: $bcgov-grey !important;
+  }
+  &.v-stepper--alt-labels {
+    .v-stepper__step {
+      padding-left: 0;
+      padding-right: 0;
+      .v-stepper__label {
+        color: $bcgov-font;
+        font-size: .75em;
+      }
+      &.v-stepper__step--active {
+        .v-stepper__label{
+          color: $bcgov-font;
+          font-weight: bold;
+        }
+      }
+    }
+  }
+}
```

Comment:
Probably a to-do but this shouldn't probably be named silv in the long run -> should be generalized.